packages/dx-scheduler-core/src/plugins/month-view/computeds.js

```diff
@@ -1,10 +1,37 @@
 import moment from 'moment';
+import {
+  sortAppointments,
+  viewPredicate,
+  toPercentage,
+} from './../../utils';
+import {
+  findOverlappedAppointments,
+  adjustAppointments,
+  unwrapGroups,
+} from './../week-view/helpers';
+import {
+  sliceAppointmentByWeek,
+  getRectByDates,
+} from './helpers';
 
 const DAY_COUNT = 7;
 const WEEK_COUNT = 6;
+const MONTH_COUNT = 31;
```

Comment:
`MONTH_COUNT` sounds strange in this case. I think, `MAX_MONTH_LENGTH` or `MONTH_LENGTH` is better.